lib/api/html/functions.php

```diff
@@ -99,25 +94,24 @@ function beans_remove_output( $id ) {
  *
  * @since 1.0.0
  *
- * @param string $id               A unique string used as a reference. The $id argument may contain sub-hooks(s).
- * @param string|bool $tag         The HTML tag. If set to False or empty, the markup HTML tag will be removed but
- *                                 the actions hook will be called. If set the Null, both markup HTML tag and actions
- *                                 hooks will be removed.
+ * @param string       $id               A unique string used as a reference. The $id argument may contain sub-hooks(s).
+ * @param string|bool  $tag         The HTML tag. If set to False or empty, the markup HTML tag will be removed but
+ *                                  the actions hook will be called. If set the Null, both markup HTML tag and actions
+ *                                  hooks will be removed.
  * @param string|array $attributes Optional. Query string or array of attributes. The array key defines the
  *                                 attribute name and the array value defines the attribute value. Setting
  *                                 the array value to '' will display the attribute value as empty
  *                                 (e.g. class=""). Setting it to 'false' will only display
  *                                 the attribute name (e.g. data-example). Setting it to 'null' will not
  *                                 display anything.
- * @param mixed  $var              Optional. Additional variables passed to the functions hooked to <tt>$id</tt>.
+ * @param mixed        $var              Optional. Additional variables passed to the functions hooked to <tt>$id</tt>. // @codingStandardsIgnoreLine - Squiz.Commenting.FunctionComment.ExtraParamComment.
```

Comment:
1. Whoops, the description is not inline with the others. 
2. Add the reason why we are ignoring the rule.